### PR TITLE
added github icon and link

### PR DESCRIPTION
**Pull request by devxMani**
#216  done added

## Summary by CodeRabbit

- **New Features**
	- Added a GitHub social media link in the footer with an icon.
	- Updated the TikTok link for better accessibility.

These changes enhance the social media presence of the application, providing users with more options to connect.

### CODE REVIEW

File: frontend/src/components/Shared/footer/Content.jsx [hunk 1]
```diff
@@ -96,9 +95,14 @@ const Nav = () => {
     },
     {
       name: 'Tiktok',
-      link: 'https://www.tiktok.com/@sipnplaynycofficial?lang=en',
+      link: 'https://www.tiktok.com/@sipnplaynyc?lang=en',
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Shared/footer/Content.jsx:98]: @devxMani why did you change the user name ?

File: frontend/src/components/Shared/footer/Content.jsx [hunk 2]
```diff
@@ -96,9 +95,14 @@ const Nav = () => {
...
       icon: <FaTiktok />,
     },
+    {
+      name: 'GitHub',
+      link: 'https://github.com/devxMani/PlayCafe',
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Shared/footer/Content.jsx:103]: add this link here:-
https://github.com/RamakrushnaBiswal/PlayCafe

File: frontend/src/components/Shared/footer/Content.jsx (hunk 1 above)
Comment (devxMani) [reply frontend/src/components/Shared/footer/Content.jsx:98]: i think by mistake hogya hoga

Comment (RamakrushnaBiswal) [reply frontend/src/components/Shared/footer/Content.jsx:98]: ok buddy thik karle 😁

File: frontend/src/components/Shared/footer/Content.jsx (hunk 2 above)
Comment (devxMani) [reply frontend/src/components/Shared/footer/Content.jsx]: @RamakrushnaBiswal done added

**Review by RamakrushnaBiswal — APPROVED** [review]
cool

**Pull request merged**